### PR TITLE
Update 'create a new file'

### LINKED ISSUE #29: “Create a new file” suggests two totally different commands
There are two alternative commands suggested:

```bash
touch 'new file'
# or
> 'new file'
```

While both commands are OK for creating files, the ultimate meaning of them is different: e. g. if the file already exists, the former would just update the access timestamp on it, while the later will also remove its contents. It is probably worth noting, especially since this guide is aimed at the beginners.

### PR DESCRIPTION
**Pull request by Berkmann18**
Fix #29

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -15,38 +15,40 @@ As a computer expert, we want to be more efficient and do our jobs better. We kn
...
-1. [force quit a program](#force-quit-a-program)
-1. [check server response](#check-server-response)
-1. [view content of a file](#view-content-of-a-file)
-1. [search for a text](#search-for-a-text)
-1. [view an image](#view-an-image)
-1. [show disk size](#show-disk-size)
-1. [check performance of your computer](#check-performance-of-your-computer)
-1. [Quick tips](#quick-tips)
-1. [Hotkeys](#hotkeys)
+- [You Don't Need GUI](#you-dont-need-gui)
```
Comment (stevemao) [inline readme.md]: Thanks for this PR. I'm not sure why this is better?

Comment (Berkmann18) [reply readme.md]: It seems that the Markdown linter I have on VSCode changed that :s, no idea how that happened.

File: readme.md [hunk 2]
```diff
@@ -15,38 +15,39 @@ As a computer expert, we want to be more efficient and do our jobs better. We kn
...
-1. [check server response](#check-server-response)
-1. [view content of a file](#view-content-of-a-file)
-1. [search for a text](#search-for-a-text)
-1. [view an image](#view-an-image)
-1. [show disk size](#show-disk-size)
-1. [check performance of your computer](#check-performance-of-your-computer)
-1. [Quick tips](#quick-tips)
-1. [Hotkeys](#hotkeys)
+1. [Quick links](#quick-links)
+2. [copy a file](#copy-a-file)
```
Comment (stevemao) [inline readme.md]: No need to change these.

Comment (Berkmann18) [reply readme.md]: Fixed.

**Comment by Berkmann18** [comment]
No problem 😁.

**Pull request merged**